lib/erb-helper.coffee

```diff
@@ -9,27 +9,34 @@ module.exports =
       'erb-helper:output'  : => @output()
       'erb-helper:eval'    : => @eval()
       'erb-helper:comment' : => @comment()
+      'erb-helper:end'     : => @end()
     }
 
   deactivate: ->
     @subscriptions.dispose()
 
   output: ->
-    @insertTag('<%=  %>')
+    @insertTag('<%=  %>', false)
 
   eval: ->
-    @insertTag('<%  %>')
+    @insertTag('<%  %>', false)
 
   comment: ->
-    @insertTag('<%#  %>')
+    @insertTag('<%#  %>', false)
 
-  insertTag: (tag) ->
+  end: ->
+    @insertTag('<% end %>', true)
+
+  insertTag: (tag, isBlockLevel) ->
```

Comment:
With coffeescript you can have a default argument like `insertTag: (tag, isBlockLevel = false) ->`. I would like to see that here so the second argument can be removed from `output`, `eval`, and `comment`